distribution/docs/src/main/resources/content/_securing/auditing.adoc

```diff
@@ -47,6 +47,12 @@ In the event the system is unable to write to the `security.log` file, ${brandin
 * edit `${home_directory}/etc/org.ops4j.pax.logging.cfg`
 ** uncomment the line (remove the `#` from the beginning of the line) for `log4j2` (`org.ops4j.pax.logging.log4j2.config.file = ${karaf.etc}/log4j2.config.xml`)
 ** delete all subsequent lines
+
+If you want to change the location of your systems security backup log from the default: `${home_directory}/data/log/securityBackup.log`, follow the next two steps:
+
+* edit `${home_directory}/security/configurations.policy`
+** find `//  permission java.io.FilePermission "<PATH>", "read,write";` under Security-Hardening
+** uncomment that line and change the value of <PATH> to the absolute path of <NEW_FILE_NAME> (described below)
```

Comment:
❓ Should this be the file name or just the dir? I might be misunderstanding but if this file rolls over, won't we have the same problem?

Reply:
That might be right. Probably should grant read/write to the fallback directory and not the specific file.

Reply:
One way to test what might happen would be to fill up the log file. 
You could do this by manually filling up the file from command line. This might be useful: https://www.skorks.com/2010/03/how-to-quickly-generate-a-large-file-on-the-command-line-with-linux/

Reply:
You were right on this Vina,
Thanks for the link to help test it